File: Practica 10/Python/Triangulo.py
from Figura import Figura

class Triangulo(Figura):
    def __init__(self, color: str, base: float, height: float, side1: float, side2: float):
        super().__init__(color)
        self.base = base
        self.height = height
        self.side1 = side1
        self.side2 = side2
    
    def perimetro(self) -> float:
        return self.base + self.side1 + self.side2
    
    def area(self) -> float:
        return 0.5 * self.base * self.height

File: Practica 10/Java/Triangulo.java
package Java;

public class Triangulo extends Figura {
    private double base, height, side1, side2;

    public Triangulo(String color, double base, double height, double side1, double side2) {
        super(color);
        this.base = base;
        this.height = height;
        this.side1 = side1;
        this.side2 = side2;
    }

    @Override
    public double perimetro() {
        return base + side1 + side2;
    }

    @Override
    public double area() {
        return 0.5 * base * height;
    }
}